frontend/packages/dev-console/src/components/import/image-search/ImageStreamTagDropdown.tsx

```diff
@@ -51,6 +59,7 @@ const ImageStreamTagDropdown: React.FC = () => {
           setFieldValue('isi.tag', selectedTag);
           setFieldValue('isi.ports', ports);
           setFieldValue('image.ports', ports);
+          setFieldValue('labels', imgStreamLabels);
```

Comment:
I dont see `labels` prop added anywhere in the form values also I dont see this form values prop being used anywhere in the code you have added.